input/fsh/EX_TreatmentLocation.fsh

```diff
@@ -0,0 +1,12 @@
+
+Instance: TreatmentLocation-1
+InstanceOf: USCoreLocation
+Title: "Treatment Location 1"
+Description: "An example treatment location."
+Usage: #example
+* id = "Treatment-Location-1"
+* meta.security = http://terminology.hl7.org/CodeSystem/v3-ActReason#HTEST
+* type = v3-RoleCode#RADDX "Radiology diagnostics or therapeutics unit"
+* name = "Joe's Radiotherapy Clinic"
+* address
+  * state = "MN"
```

Comment:
Should we invent a full address here as the requirement was to cover Institution name and address?

Reply:
added a city/state/street address